components/admin/requests/SelectedPermitHolderCard.tsx

```diff
@@ -0,0 +1,147 @@
+  Tooltip,
+  useClipboard,
+  Flex,
+  Wrap,
+  Badge,
+} from '@chakra-ui/react'; // Chakra UI
+import { ApplicantData } from '@tools/pages/admin/permit-holders/types'; // Applicant data type
+import { formatDate } from '@lib/utils/format'; // Date formatter util
+
+type SelectedPermitHolderCardProps = {
+  readonly applicant: ApplicantData;
+};
+
+export default function SelectedPermitHolderCard(props: SelectedPermitHolderCardProps) {
+  const { applicant } = props;
+  const { hasCopied, onCopy } = useClipboard(applicant?.email ? applicant?.email : '');
+
+  return (
+    <GridItem
+      display="flex"
+      flexDirection="column"
+      alignItems="flex-start"
+      padding="20px 24px 24px"
+      background="white"
+      border="1px solid"
+      borderColor="border.secondary"
+      boxSizing="border-box"
+      borderRadius="8px"
+      textAlign="left"
+    >
+      <Flex marginTop={5} alignItems="center">
+        <Box>
+          <Flex alignItems="center">
+            <Text textStyle="display-large" as="h1" marginRight={3}>
+              {`${applicant.firstName} ${applicant.lastName}`}
+            </Text>
+            {applicant.status && (
+              <Wrap>
+                <Badge variant={applicant.status}>{applicant.status}</Badge>
+              </Wrap>
+            )}
+          </Flex>
+          <HStack spacing={3} marginTop={3}>
+            <Text textStyle="caption" as="p">
+              ID: #{applicant.rcdUserId}
+            </Text>
+          </HStack>
+        </Box>
+      </Flex>
+
+      <Divider mt="24px" />
+
+      <VStack spacing="12px" pt="24px" align="left">
+        <HStack spacing="12px">
+          <Box>
+            <Text as="h4" textStyle="body-bold">
+              Personal Information
+            </Text>
+          </Box>
+        </HStack>
+        <Box>
+          <Text as="p" textStyle="body-regular">
+            Date of Birth: {formatDate(applicant.dateOfBirth)}
+          </Text>
+        </Box>
+        <Box>
+          <Text as="p" textStyle="body-regular">
+            Gender: {applicant.gender.toLowerCase().replace(/^\w/, c => c.toUpperCase())}
+          </Text>
+        </Box>
+      </VStack>
+
+      <Divider mt="24px" />
+
+      <VStack spacing="12px" pt="24px" align="left">
+        <HStack spacing="12px">
+          <Box>
+            <Text as="h4" textStyle="body-bold">
+              Contact Information
+            </Text>
+          </Box>
+        </HStack>
+        <Box>
+          <Tooltip
+            hasArrow
+            closeOnClick={false}
+            label={hasCopied ? 'Copied to clipboard' : 'Click to copy address'}
+            placement="top"
+            bg="background.grayHover"
+            color="black"
+          >
+            <Link
+              textStyle="body-regular"
+              color="primary"
+              textDecoration="underline"
+              onClick={onCopy}
+            >
+              {applicant.email}
+            </Link>
```

Comment:
`Link` missing `href`. Also use `next/link`'s `Link` component, not Chakra's

Reply:
Does it need a href? It is just for copying the email. Not sending anyone anywhere?

Reply:
Also, I think I need to use chakra UI's link because of this, since next.js link requires href.

Reply:
I see, we can leave this for now then